src/App.tsx

```diff
@@ -0,0 +1,27 @@
+import "./styles.css";
+import { useState } from "react";
+import Sidebar from "./Components/Sidebar/Sidebar";
+import Body from "./Components/Body/Body";
+import Header from "./Components/Header/Header";
```

Comment:
Follow import order as we discussed.
libs -> components -> hooks -> helpers -> types -> constants